### PR TITLE
Migrate to aws-sdk-go-v2

### PR DESCRIPTION
**Pull request by at-wat**
requires https://github.com/at-wat/mqtt-go/pull/175
will bump the major version to v6

### CODE REVIEW

**Comment by at-wat** [comment]
@seqsense/cloud_backend please take a look along with https://github.com/at-wat/mqtt-go/pull/175

File: tunnel/server/json.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	"reflect"
+	"strings"
+	"unicode"
+)
+
+type response struct {
+	value interface{}
+}
+
+func (j *response) MarshalJSON() ([]byte, error) {
```
Comment (kamatama41) [inline tunnel/server/json.go:28]: Why this func is needed?
Is just using `json.Marshal` insufficient?

Comment (kamatama41) [reply tunnel/server/json.go:28]: Ah, it's for making keys lower camel case, right?

**Review by kamatama41 — APPROVED** [review]
🆗

File: tunnel/server/json.go (hunk 1 above)
Comment (at-wat) [reply tunnel/server/json.go:28]: That's right

**Review by t-asaka — APPROVED** [review]
LGTM

**Pull request merged**